api/azure/flavors.go

```diff
@@ -0,0 +1,124 @@
+package azure
+
+import (
+	"github.com/elastisys/ck8s/api"
+)
+
+const (
+	FlavorDevelopment api.ClusterFlavor = "dev"
+	FlavorProduction  api.ClusterFlavor = "prod"
+)
+
+func Default(clusterType api.ClusterType, clusterName string) *Cluster {
+	return &Cluster{
+		config: AzureConfig{
+			BaseConfig: *api.DefaultBaseConfig(
+				clusterType,
+				api.Azure,
+				clusterName,
+			),
+			// TODO change this to azure storage
+			S3RegionAddress: "sos-ch-gva-2.exo.io",
+
+			TenantID:       "changeme",
+			SubscriptionID: "changeme",
+			Location:       "changeme",
+		},
+		secret: AzureSecret{
+			BaseSecret:   *api.DefaultBaseSecret(),
+			ClientID:     "changeme",
+			ClientSecret: "changeme",
+		},
+		tfvars: AzureTFVars{
+			PublicIngressCIDRWhitelist: []string{},
+			APIServerWhitelist:         []string{},
+			NodeportWhitelist:          []string{},
+		},
+	}
+}
+
+func Development(clusterType api.ClusterType, clusterName string) api.Cluster {
+	cluster := Default(clusterType, clusterName)
+
+	cloudProvider := NewCloudProvider()
+
+	master := api.NewMachineFactory(
+		cloudProvider,
+		api.Master,
+		// 2C-8GB-50GB
+		"Standard_D2_v3",
```

Comment:
It would be nice to have a list of common sizes in a comment, or at least a link to some documentation. :slightly_smiling_face: